sql/2021/ecommerce/pagestats_html_bydevice.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+# 13_07: Distribution of HTML kilobytes per page
+SELECT
+  _TABLE_SUFFIX AS client,
+  ROUND(APPROX_QUANTILES(bytesHtml, 1000)[OFFSET(100)] / 1024, 2) AS p10,
```

Comment:
Not necessary but it'd be nice to rewrite this in the style of the following query with `UNNEST([10, 25, 50, 75, 90, 100]) AS percentile`. It cuts down on redundancy. Also it's nice to include the 100th percentile as a fun stat.

Reply:
I fixed this as well